client/mainmenu/CPrologEpilogVideo.cpp

```diff
@@ -102,3 +103,14 @@ void CPrologEpilogVideo::clickPressed(const Point & cursorPosition)
 	if(exitCb)
 		exitCb();
 }
+
+void CPrologEpilogVideo::clickPressed(const Point & cursorPosition)
+{
+	exit();
+}
+
+void CPrologEpilogVideo::keyPressed(EShortcut key)
+{
+	if(key == EShortcut::GLOBAL_ACCEPT || key == EShortcut::GLOBAL_CANCEL)
```

Comment:
There is already key that is bound to both Enter and Escape.
Accept / Cancel are for Yes/No dialogs and similar.
```suggestion
	if(key == EShortcut::GLOBAL_RETURN)
```